endaq/device/schemata/command-response.xml

```diff
@@ -163,6 +163,8 @@
         </UIntegerElement>
 		<BinaryElement name="PingReply" id="0x5701" multiple="0" mandatory="0">Serial only. Contains the echo to a Ping command.</BinaryElement>
         <IntegerElement name="DeviceStatusCode" id="0x5801" multiple="0" mandatory="0">Serial only, mandatory for serial. int16. Negative values are errors.
+            200: Device Resetting
```

Comment:
How does `200: Device Resetting` differ from `20: Reset pending`? I think 20 is 'pending' just because it was planned as a response to the reset command, so technically it was only about to reset, not yet resetting.